src/karts/kart.cpp

```diff
@@ -118,6 +118,7 @@ Kart::Kart (const std::string& ident, unsigned int world_kart_id,
     m_max_speed            = new MaxSpeed(this);
     m_terrain_info         = new TerrainInfo();
     m_powerup              = new Powerup(this);
+    m_last_used_powerup    = new PowerupManager::PowerupType();
```

Comment:
You don't need to "new" an enum, or use a pointer to an enum, this has no reason to be on the heap

Reply:
Right, I changed the occurences everywhere when I converted it from a full Powerup type to a simple PowerupType without thinking about this. I'll remove it.